internal/executor/shell.go

```diff
@@ -61,15 +60,16 @@ func ShellCommandsAndWait(
 
 	select {
 	case <-ctx.Done():
-		handler([]byte("\nTimed out!"))
+		errorMessage := fmt.Sprintf("%v!", context.Cause(ctx))
+		handler([]byte("\n" + strings.ToUpper(errorMessage[:1]) + errorMessage[1:]))
```

Comment:
You can probably use [`cases.Title()`](https://pkg.go.dev/golang.org/x/text/cases#Title) here to skip the gory details of string manipulation.